src/pepr/uds-operator-config/templates/clusterconfig.yaml

```
@@ -0,0 +1,31 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: uds.dev/v1alpha1
+kind: ClusterConfig
+metadata:
+  name: uds-cluster-config
+  namespace: pepr-system
+  labels:
+    {{- include "uds-operator-config.labels" . | nindent 4 }}
+spec:
+  attributes:
+    clusterName: {{ .Values.cluster.attributes.clusterName | quote }}
+    tags:
+{{ toYaml .Values.cluster.attributes.clusterTags | nindent 6 }}
+  networking:
+    kubeApiCIDR: {{ .Values.cluster.networking.kubeApiCIDR | default .Values.operator.KUBEAPI_CIDR| quote }}
+    kubeNodeCIDRS:
```

This needs an update, lowercase `s`?:
```
I0709 17:06:59.720915   91671 warnings.go:110] "Warning: unknown field \"spec.networking.kubeNodeCIDRS\""
```

```suggestion
    kubeNodeCIDRs:
```